bin/zssServer.sh

```diff
@@ -41,21 +41,6 @@ then
 elif [ -e "$ZLUX_CONFIG_FILE" ]
 then
     CONFIG_FILE=$ZLUX_CONFIG_FILE
-elif [ -d "$ZWE_zowe_workspaceDirectory" ]
```

Comment:
You appear to not have CONFIG_FILE set if zss is run without app-server.
You should make it similar to app-server, like https://github.com/zowe/zlux-app-server/pull/199/files#r801080370

Reply:
I fixed it in a bit different way, because zss still can't accept yaml files directly.